release-1.8/release_notes_draft.md

```diff
@@ -37,30 +37,30 @@ please check out the [release notes guidance][] issue.
 
 ## **Major Themes**
 
-Kubernetes is developed by community members organized into
-[Special Interest Groups][]. For the 1.8 Release each SIG has provided the
-themes which guided their work.
+Kubernetes is developed by community members whose work is organized into
+[Special Interest Groups][]. For the 1.8 release, each SIG provides the
+themes that guided their work.
 
 [Special Interest Groups]: https://github.com/kubernetes/community/blob/master/sig-list.md
 
 ### SIG Apps
 
-[SIG Apps][] focuses on the Kubernetes APIs and external tools required to deploy and operate a wide variety of workloads.
+[SIG Apps][] focuses on the Kubernetes APIs and the external tools that are required to deploy and operate Kubernetes workloads.
 
-For the 1.8 release, SIG Apps moved the kubernetes workloads API to the new apps/v1beta2 group and version. This includes DaemonSet, Deployment, ReplicaSet, and StatefulSet. Though apps/v1beta2 group introduces several deprecations and behavioral changes, it provides developers with a stable and consistent API surface to build applications in Kubernetes. SIG Apps intends to promote this version to GA in a future release.
+For the 1.8 release, SIG Apps moved the Kubernetes workloads API to the new apps/v1beta2 group and version. The DaemonSet, Deployment, ReplicaSet, and StatefulSet objects are affected by this change. The new apps/v1beta2 group and version provide a stable and consistent API surface for building applications in Kubernetes. For details about deprecations and behavioral changes, see [below](#notable-features). SIG Apps intends to promote this version to GA in a future release.
 
 
 [SIG Apps]: https://github.com/kubernetes/community/tree/master/sig-apps
 
 ### SIG Auth
 
-[SIG Auth][] is responsible for Kubernetes authentication, authorization, and
+[SIG Auth][] is responsible for Kubernetes authentication and authorization, and for
 cluster security policies.
 
-For the 1.8 release SIG Auth focused on stablizing existing features introduced
-in previous releases. RBAC was promoted to v1 and advanced auditing was promoted
-to beta. Encryption of resources at rest, which remained alpha, began exploring
-integrations with external Key Management Systems.
+For the 1.8 release, SIG Auth focused on stablizing existing features that were introduced
+in previous releases. RBAC was moved from beta to v1, and advanced auditing was moved from alpha
+to beta. Encryption of resources at rest remained at alpha, and the SIG began exploring
```

Comment:
what does it mean by `resource at rest`

Reply:
no it means encryption of resources stored on disk , vs ecnryption of data in motion like when its being transported.

Reply:
clarified